### PR TITLE
1st time explore screen

### PR DESCRIPTION
**Pull request by dev-tronghieu**

### CODE REVIEW

File: apps/extension/src/screens/Dashboard/ChooseLayout/ChooseLayoutCard.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import { Image } from 'react-native';
+import { Text, TextBlack, View } from '@walless/ui';
+
+interface Props {
+	name: string;
+	description: string;
+	love?: number;
+	activeUsers?: number;
```
Comment (fluctlight-kayaba) [inline apps/extension/src/screens/Dashboard/ChooseLayout/ChooseLayoutCard.tsx:8]: consider put those field under an interface named e.g: `interface WalletLayout {...}`, and then:
```typescript
import type { WalletLayout } from 'url-to-types'

interface Props {
  item: WalletLayout;
}
```

It is most likely `WalletLayout` will be defined via GraphQL api (or other kind of api interface), client-side/front-end use generated Typescript interface from the api.

File: apps/extension/src/screens/Dashboard/ChooseLayout/ChooseLayoutCard.tsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
+import { Image } from 'react-native';
+import { Text, TextBlack, View } from '@walless/ui';
```
Comment (fluctlight-kayaba) [inline apps/extension/src/screens/Dashboard/ChooseLayout/ChooseLayoutCard.tsx:2]: Also import `Image` from `@walless/ui`.

File: apps/extension/src/screens/Dashboard/ChooseLayout/ChooseLayoutCard.tsx [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+	description: string;
+	love?: number;
+	activeUsers?: number;
+}
+
+const ChooseLayoutCard = (props: Props) => {
+	return (
+		<View className="bg-[#f4f4f4] w-full rounded-b">
+			<View className="h-16 bg-light-gray rounded-t">
+				<Image source={require('./card-image.png')} />
```
Comment (fluctlight-kayaba) [inline apps/extension/src/screens/Dashboard/ChooseLayout/ChooseLayoutCard.tsx:15]: it is more convenient to put image files under `assets/img` folder, and declare asset resource under [utils/config.js](https://github.com/cocrafts/walless/blob/main/apps/extension/src/utils/config.ts).

and then, use following snippet:
```typescript
import { resources } from 'utils/config';

<Image source={resources.path.to.resource}/>
```

File: apps/extension/src/screens/Dashboard/index.tsx [hunk 4]
```diff
@@ -1,17 +1,12 @@
...
 export const HomeScreen: FC = () => {
 	return (
 		<DashboardLayout contentContainerClass="bg-gradient-to-b from-color-5 to-color-6 flex-1">
-			<Text className="text-white text-5xl mb-3">COMING SOON!</Text>
-			<Text>
-				<Text className="text-gray-400">Created with ❤️ by Metacraft</Text>
-			</Text>
-			<Link to="login">Login</Link>
+			<ChooseLayout />
 		</DashboardLayout>
```
Comment (fluctlight-kayaba) [inline apps/extension/src/screens/Dashboard/index.tsx:10]: Use self-closing syntax for empty Tag.
```typescript
<DashboardLayout ... />
```

File: packages/ui/components/IconSearch.tsx [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+				strokeLinejoin="round"
+			/>
+			<path
+				d="M17 10a7 7 0 1 1-14 0 7 7 0 0 1 14 0Z"
+				stroke={props.color || '#000'}
+				strokeWidth={2}
+			/>
+		</svg>
+	);
+};
```
Comment (fluctlight-kayaba) [inline packages/ui/components/IconSearch.tsx:37]: `1.` Props already declared via `FC<Props>`, `props: Props` could be just `props`.
`2.` Think about destruct props: `const IconSearch: FC<Props> = ({ size, color }) => { ... };`

File: apps/extension/src/screens/Dashboard/index.tsx (hunk 4 above)
Comment (dev-tronghieu) [reply apps/extension/src/screens/Dashboard/index.tsx:10]: Hi leader, I don't get the problem. There is a `<ChooseLayout />` (with self-closing syntax already) inside the `<DashboardLayout> <ChooseLayout /> </DashboardLayout>`.

**Pull request closed without merging**